### PR TITLE
Feat/multer s3

### PR DESCRIPTION
**Pull request by MikeGonzaBar**

### CODE REVIEW

**Comment by natxsglez** [comment]
Please add stylings to the elements, the pictures will look strange without this. I can approve the PR after that

File: frontend/src/app/common/components/update-password/update-password.component.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+})
+export class UpdatePasswordComponent implements OnInit {
+  userId: string = '';
+  inputPassword: string = '';
+  inputPasswordConfirm: string = '';
+  constructor(private userService: UserService, private authService: AuthService) {}
+
+  updatePassword(event: Event) {
+    if (this.inputPassword == '' || this.inputPasswordConfirm == '') {
+      console.log('An input is empty');
```
Comment (natxsglez) [inline frontend/src/app/common/components/update-password/update-password.component.ts:18]: Delete console.log()

**Review by natxsglez — APPROVED** [review]
LGTM :D

**Review by kevinglezgmz — APPROVED** [review]
LGTM ;)

**Pull request merged**